process/interface.go

```diff
@@ -416,13 +416,23 @@ type RewardsHandler interface {
 	IsInterfaceNil() bool
 }
 
-// FeeHandler will return information about fees
+// FeeHandler is able to perform some economics calculation on a provided transaction
 type FeeHandler interface {
-	MinGasPrice() uint64
-	MinGasLimit() uint64
+	SetMinGasPrice(minGasPrice uint64)
+	SetMinGasLimit(minGasLimit uint64)
+	ComputeGasLimit(tx TransactionWithFeeHandler) uint64
+	ComputeFee(tx TransactionWithFeeHandler) *big.Int
+	CheckTxHandler(tx TransactionWithFeeHandler) error
```

Comment:
might rename the function, CheckTxValuesValidity or something similar ?

Reply:
CheckValidityTxValues